jobsdb/readonly_jobsdb.go

```diff
@@ -35,9 +35,10 @@ type ReadonlyJobsDB interface {
 }
 
 type ReadonlyHandleT struct {
-	DbHandle    *sql.DB
-	tablePrefix string
-	logger      logger.Logger
+	DbHandle           *sql.DB
+	tablePrefix        string
+	logger             logger.Logger
+	maxOpenConnections int
```

Comment:
don't really need it to be a struct field, this would suffice

```
jd.DbHandle.SetMaxOpenConns(config.GetInt("ReadonlyJobsDB." + jd.tablePrefix + "." + "maxOpenConnections", config.GetInt("ReadonlyJobsDB.maxOpenConnections", 5)))```

Reply:
Agreed!